Tabloid/client/src/components/CategoryList.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
+import Category from "./Category";
+import { getAllCategories } from "../modules/categoryManager";
+
+const CategoryList = () => {
+  const [categories, setCategories] = useState([]);
+  
+
+  const getCategories = () => {
+    getAllCategories().then((categories) => setCategories(categories));
+  };
+
+
+  useEffect(() => {
+    getCategories();
+  }, []);
+
+  console.log(categories)
```

Comment:
I can see this console.log being useful for testing purposes but unnecessary for deployment.